.github/workflows/cd.yml

```diff
@@ -0,0 +1,98 @@
+            libtorrent: 2.0.5
+          - arch: x86
+            wheel_arch: win32
+            pythonVersion: 3.7
+            gvsbuild: "https://github.com/doadin/gvsbuild/releases/download/1-7-2022-5/gvsbuild-py3.7-vs16-x86.zip"
+            libtorrent: 2.0.5
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.pythonVersion}}
+          architecture: ${{ matrix.arch }}
+
+      - name: Cache pip
+        uses: actions/cache@v2
+        with:
+          path: '%LOCALAPPDATA%\pip\Cache'
+          # Look to see if there is a cache hit for the corresponding requirements file
+          key: ${{ runner.os }}-pip-${{ hashFiles('tox.ini', 'setup.py', 'requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+            ${{ runner.os }}-
+
+      - name: Install dependencies
+        env:
+          PYTHONVERSION: ${{ matrix.pythonVersion}}
+          WHEEL_ARCH: ${{ matrix.wheel_arch }}
+          ARCH: ${{ matrix.arch }}
+          GVSBUILD: ${{ matrix.gvsbuild }}
+          LIBTORRENT: ${{ matrix.libtorrent }}
+        run: |
+          $WebClient = New-Object System.Net.WebClient
+          $WebClient.DownloadFile($env:GVSBUILD,"C:\GTK.zip")
+          7z x C:\GTK.zip -oc:\GTK
+          $env:Path = "C:\GTK\release;C:\GTK\release\bin;C:\GTK\release\lib;$env:Path"
+          python -m pip install --upgrade pip wheel
+          $pycairopath = Get-Childitem –Path "C:\GTK\release\python\" -Include pycairo*.whl -File -Recurse -ErrorAction SilentlyContinue | select -expand FullName
+          $PyGObjectpath = Get-Childitem –Path "C:\GTK\release\python\" -Include PyGObject*.whl -File -Recurse -ErrorAction SilentlyContinue | select -expand FullName
+          pip install $pycairopath
+          pip install $PyGObjectpath
+          python -m pip install libtorrent==$env:LIBTORRENT
+          pip install -r requirements.txt
+          pip install pyinstaller
+
+      - name: Install Deluge
+        run: |
+          python setup.py clean
+          python setup.py build
+          python setup.py install
+
+      - name: Freeze Deluge
+        run: |
+          #gci env:* | sort-object name
+          $env:Path = "C:\GTK\release;C:\GTK\release\bin;C:\GTK\release\lib;$env:Path"
+          pyinstaller --clean $env:GITHUB_WORKSPACE\packaging\win\delugewin.spec --distpath $env:GITHUB_WORKSPACE\packaging\win\freeze
+          Get-ChildItem $env:GITHUB_WORKSPACE\packaging\win\freeze\* -Include requires.txt -Recurse | Remove-Item
+
+      - name: Make Deluge Installer
+        env:
+          ARCH: ${{ matrix.arch }}
+        run: |
+          Set-Location -Path $env:GITHUB_WORKSPACE\packaging\win
```

Comment:
Can simplify to using run command `working-dir`keyword: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsrun

Reply:
Set-Location line can be removed then